src/app/public/modules/inline-delete/inline-delete.component.ts

```diff
@@ -0,0 +1,57 @@
+import {
+  AnimationEvent
+} from '@angular/animations';
+
+import {
+  Component,
+  Input,
+  ChangeDetectorRef,
+  Output,
+  EventEmitter
+} from '@angular/core';
+
+import {
+  skyAnimationEmerge
+} from '@skyux/animations';
+
+import {
+  SkyInlineDeleteType
+ } from './inline-delete-type';
+
+@Component({
+  selector: 'sky-inline-delete',
+  styleUrls: ['./inline-delete.component.scss'],
+  templateUrl: './inline-delete.component.html',
+  animations: [
+    skyAnimationEmerge
+  ]
+})
+export class SkyInlineDeleteComponent {
+
+  public animationState: string = 'open';
```

Comment:
This property should come after the `Input`/`Output`s.

Reply:
Done